pages/projects/[slug].tsx

```diff
@@ -0,0 +1,107 @@
+import { PreviewSuspense } from '@sanity/preview-kit'
+import { ProjectPage } from 'components/pages/project/ProjectPage'
+import { ProjectPreview } from 'components/pages/project/ProjectPreview'
```

Comment:
When using `/pages` we're no longer getting `React.lazy` for free on preview components that call the `usePreview`. 
We should make sure to add them so that our starters don't load any JS that only preview mode needs if preview mode isn't enabled. Preview mode loads a ton of stuff, an entire EventSource polyfill, the groq js parser and more.
I think next-sanity documents the conventions for this pattern with PreviewSuspense 🤔

Reply:
Gocha! Fixing!